spec/lib/bank_account/checkings.rb

```diff
@@ -0,0 +1,37 @@
+require 'bank_account'
```

Comment:
@knguyen0710 It looks like this mostly the same as `checking_spec.rb`, so I'm going to look at that one instead. If you aren't using this file, you might want to remove it!